controllers/mirrorpeersecret_controller.go

```diff
@@ -58,47 +59,75 @@ type MirrorPeerSecretReconciler struct {
 // Reconcile standard reconcile function for MirrorPeerSecret controller
 func (r *MirrorPeerSecretReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	// reconcile for 'Secrets' (source or destination)
-	return ctrl.Result{}, mirrorPeerSecretReconcile(ctx, r.Client, req)
+	return mirrorPeerSecretReconcile(ctx, r.Client, req)
 }
 
-func mirrorPeerSecretReconcile(ctx context.Context, rc client.Client, req ctrl.Request) error {
+func updateSecretWithHubRecoveryLabel(ctx context.Context, rc client.Client, peerSecret corev1.Secret) error {
```

Comment:
Is this updateSecretWithHubRecoveryLabel will take of for both internal and external cluster secrets?

Reply:
external cluster secret is labelled as "INTERNAL" and converged mode cluster is indicated with "BLUE" secret label. So it should be able to handle both